### DISCUSSION TITLE
Light background by default

### DISCUSSION THREAD

**Discussion by slimooo** [retypeapp/retype#19] [Q&A] [answered]
Hello,

thanks for your work ! It's possible to choose white background by default ?

Thank you

**Comment by geoffreymcgill** [comment, marked as answer]
Hi. By default, Retype switches automatically between the `light` and `dark` themes depending on the time of the day.

If it's "day time" for you right now, but you are seeing the `dark` theme, it's likely because the Theme switcher button (top right corner) has been clicked and the app explicitly switched to the `dark` theme. You can click the button again to switch to the `light` theme.

<img width="296" alt="Screen Shot 2021-05-27 at 11 15 03 AM" src="https://user-images.githubusercontent.com/62210/119869057-0a9c5700-bedd-11eb-9f61-2b7e4cb21890.png">

**Reply by fabriciomurta** [reply]
Notice on Windows there's no such night-day auto switching. So on windows it would always _default_ to the option chosen in desktop preferences.

Once the _Theme switcher button_  is hit, its last setting is remembered regardless of OS setting.

**Reply by slimooo** [reply]
I test it day and night, on Android 11 with firefox, and on mac os 11.4 same result : dark mode. In the two cases I had activate dark mode option in the OS config.

**Comment by slimooo** [comment]
👍🏽
